LAKMobile/src/screens/ListJobs.tsx

```diff
@@ -16,23 +17,22 @@ const ADD_PICKER_OPTIONS: JobTypePickerOption[] = ['Current Jobs', 'Completed Jo
 
 const FIND_PICKER_OPTIONS: JobTypePickerOption[] = ['Your Jobs', 'Finished Jobs'];
 
-const LIST_MODES: ListJobsModes[] = ['Add', 'Find'];
-
 interface ListJobsProps {
   navigation: any;
   mode: ListJobsModes;
 }
 
 export function ListJobs({ navigation, mode }: ListJobsProps) {
-  const [jobListType, setJobListType] = useState<JobTypePickerOption>('Current Jobs');
+  const [jobListType, setJobListType] =
+    mode === 'Add'
+      ? useState<JobTypePickerOption>('Current Jobs')
+      : useState<JobTypePickerOption>('Your Jobs');
```

Comment:
I think it's a little weird to pick between two useState options. We can move the ternary statement as `useState<JobTypePickerOption>(mode === 'Add' ? 'Current Jobs' : 'Your Jobs');`